src/utils.rs

```diff
@@ -87,6 +87,10 @@ active boot switches: -d:release\n",
             stdout: String::from("4.10.0"),
             stderr: String::default(),
         }),
+        "perl -e print $^V;" => Some(CommandOutput {
```

Comment:
This won't work cross-platform, in fact, it didn't on my Android. To make it work use the following form: `perl -e 'printf "%vd", $^V'`

Reply:
ok i've used  perl -e 'printf q#%vd#,$^V;' to try to avoid any double escape condition

Reply:
Perfect 👍